mappings/net/minecraft/client/render/chunk/ChunkBuilder.mapping

```diff
@@ -3,20 +3,35 @@ CLASS net/minecraft/unmapped/C_cifoaixd net/minecraft/client/render/chunk/ChunkB
 	FIELD f_bwqkxojd queuedTaskCount I
 	FIELD f_dcgdahlo LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD f_dlijlhna uploadQueue Ljava/util/Queue;
+	FIELD f_fzsfhpbo chunksToBuild Ljava/util/Queue;
 	FIELD f_inqsqser POSITION_COLOR_TEXTURE_LIGHT_NORMAL Lnet/minecraft/unmapped/C_rnldvdpe;
 	FIELD f_nlltickx world Lnet/minecraft/unmapped/C_ghdnlrrw;
 	FIELD f_ozkutnxz cameraPosition Lnet/minecraft/unmapped/C_vgpupfxx;
 	FIELD f_qrrzdkwd mailbox Lnet/minecraft/unmapped/C_bhuwdyga;
+	FIELD f_rgwkezae highPriorityQuota I
 	FIELD f_rmtgpazv worldRenderer Lnet/minecraft/unmapped/C_sfkkabhx;
 	FIELD f_rxdlwkdw bufferCount I
+	FIELD f_shhksbkx MAX_WORKER_THREADS_FOR_32_BIT_SYSTEMS I
```

Comment:
This could be shortened? Maybe something like `32_BIT_MAX_WORKER_THREADS`

Reply:
```suggestion
	FIELD f_shhksbkx 32_BIT_MAX_WORKER_THREADS I
```